cmd/bitxhub/client/node.go

```diff
@@ -39,3 +39,30 @@ func getValidators(ctx *cli.Context) error {
 
 	return nil
 }
+
+func delVPNodeCMD() cli.Command {
+	return cli.Command{
+		Name:   "delVPNode",
+		Usage:  "delete a vp node",
+		Action: delVPNode,
+	}
+}
+
+func delVPNode(ctx *cli.Context) error {
+	if ctx.NArg() < 1 {
+		return fmt.Errorf("please input pid")
+	}
+
+	url, err := getURL(ctx, "delvpnode/"+ctx.Args().Get(0))
+	if err != nil {
+		return err
+	}
+
+	data, err := httpGet(url)
```

Comment:
Maybe more reasonable to use "post" to delete a node?

Reply:
fixed